luis-hurtado/tarea-3/spongebobMeme.js

```diff
@@ -0,0 +1,14 @@
+function spongeMeme(text) {
+       let sentence = text.split('');
+       let retardifier = '';
+        for (let i = 0; i < sentence.length; i++) {
```

Comment:
@luishurtado-globant , you should use modern approach like forEach, map or reduce also you should be able to make this a one line solution